templates/index.html

```diff
@@ -301,7 +301,7 @@ <h1>Azul Payment Demo</h1>
             try {
                 showResult({ message: "Iniciando proceso de pago..." }, 'processing');
                 
-                const response = await fetch('/process-payment', {
+                const response = await fetch('/process-hold', {
```

Comment:
Here it indicate is a `process-hold`, but above and below it indicate it's a `payment` not a `hold`